cpp/tests/utilities/column_utilities.cu

```diff
@@ -33,18 +33,33 @@
 namespace cudf {
 namespace test {
 
-// Property equality
-void expect_column_properties_equal(cudf::column_view lhs, cudf::column_view rhs) {
+namespace {   // anonymous
+
+// check for semantic property equality.  the difference between this and 
+// expect_column_properties_equal() is that the nullability of columns doesn't
+// affect their effective equality for the purposes of expect_columns_equal().
+//
+// expect_column_properties_equal() adds on the additional nullable() check
+// for cases where explicit full equality checking is desired.
+void expect_column_properties_equal_semantic(cudf::column_view lhs, cudf::column_view rhs)
```

Comment:
BTW, as part of this PR can you change the lhs and rhs params to `column_view const&`? I have this change in my current rolling_windows PR but it is a while out.

Reply:
Will do.